### PR TITLE
Add fromBase64 support on convert

### PR DESCRIPTION
**Pull request by aq17**
Part of `pu/pu` #[10726](https://github.com/pulumi/pulumi/issues/10726)

### CODE REVIEW

**Comment by aq17** [comment]
Part of `pu/pu` #[10726](https://github.com/pulumi/pulumi/issues/10726)

File: pkg/pulumiyaml/codegen/convert.go [hunk 1]
```diff
@@ -66,6 +66,7 @@ func ConvertTemplateIL(template *ast.TemplateDecl, loader schema.ReferenceLoader
 		return "", diags, nil
 	}
 	programText := fmt.Sprintf("%v", templateBody)
+	fmt.Println(programText)
```
Comment (AaronFriel) [inline pkg/pulumiyaml/codegen/convert.go:69]: Debug comments to remove?

File: pkg/pulumiyaml/codegen/gen_program.go [hunk 2]
```diff
@@ -518,6 +518,7 @@ func (g *generator) expr(e model.Expression) syn.Node {
 		}
 
 	case *model.FunctionCallExpression:
+		fmt.Println("functionCallExpression")
```
Comment (AaronFriel) [inline pkg/pulumiyaml/codegen/gen_program.go:521]: Same

File: pkg/pulumiyaml/codegen/gen_program.go [hunk 3]
```diff
@@ -646,6 +647,7 @@ func (g *generator) genLocalVariable(n *pcl.LocalVariable) {
 }
 
 func (g *generator) function(f *model.FunctionCallExpression) syn.Node {
+	fmt.Println(f.Name)
```
Comment (AaronFriel) [inline pkg/pulumiyaml/codegen/gen_program.go:650]: Same

**Review by AaronFriel — COMMENTED** [review]
Looks good, though we should clean up the println debugging.

**Review by AaronFriel — APPROVED** [review]
lgtm

**Pull request merged**